DB/src/main/java/io/deephaven/db/v2/parquet/ParquetReaderUtil.java

```diff
@@ -346,6 +347,8 @@ public static void readParquetSchema(final String filePath, final ColumnDefiniti
                         }
                         colDefConsumer.accept(colName, byte[].class, byte.class, isGrouping, codecName, codecArgs);
                         break;
+                    case INT96:
```

Comment:
I think it would be nice to put the `INT96` case after the `INT64` case in this file and in ROPTL.